src/deepsparse/v2/schedulers/scheduler.py

```diff
@@ -52,6 +53,22 @@ def submit(
             **kwargs,
         )
 
+    def can_map(self, *args):
+        """
+        args containing list of inputs to be used for each worker. This function if we
+        have sufficient workes available
+        """
+        if len(args[0]) <= self._threadpool._max_workers:
```

Comment:
don't need this check here - definitely can have items in same split that are queued up